src/conn/conn_tiered.c

```diff
@@ -375,6 +522,7 @@ __wt_tiered_storage_create(WT_SESSION_IMPL *session, const char *cfg[], bool rec
 
     WT_ERR(__wt_open_internal_session(conn, "storage-server", true, 0, 0, &conn->tiered_session));
     session = conn->tiered_session;
+    WT_ERR(__wt_txn_reconfigure(session, "isolation=read-uncommitted"));
```

Comment:
Do we need to set this back at the end of the function?  That is, is the session we are reconfiguring going to be used for other (unrelated) initialization?